src/dialogs/permissions.tsx

```diff
@@ -255,13 +269,32 @@ const EditPermissionsModal = ({ dialogResult, selected, path } : {
         return a.name.localeCompare(b.name);
     }
 
+    let description;
+    if (items.length === 1) {
+        description = (selected.type) ? inode_types[selected.type] : _("Missing type");
+    } else {
+        description = (
+            <ExpandableSection
+              truncateMaxLines={1}
```

Comment:
Hm, so we don't have much control over that "…", i.e. how to add a space to it? Does it have some special CSS class so that we can fix it with CSS?